app/assets/stylesheets/views/_organisations.scss

```diff
@@ -1,24 +1,50 @@
 .organisations {
-  .organisations__section {
+  .organisations-list__section {
     margin-bottom: $gutter * 2;
 
     .department-count {
       @include bold-80;
     }
 
-    .organisations__logo-wrapper {
+    .organisations-list__item {
       list-style-type: none;
-      padding-bottom: $gutter-two-thirds;
-      margin-bottom: $gutter-two-thirds;
+      padding-bottom: $gutter-half;
+      margin-bottom: $gutter-half;
       border-bottom: 1px solid $grey-2;
 
-      .organisation__works-with {
+      .organisation-list__item-title {
+        @include core-19;
+        display: inline-block;
+        margin-right: $gutter-one-third;
+      }
+
+      .organisation-list__item-context {
+        color: $grey-1;
+      }
+
+      .organisation-list__item-works-with {
         @include core-16;
-        margin-top: $gutter-half;
+        display: inline-block;
+        margin-top: $gutter-one-third;
+        margin-right: $gutter-two-thirds;
+      }
+
+      .organisation-list__works-with {
+        background-color: $panel-colour;
+        margin-top: $gutter-one-third;
+        padding: $gutter-one-third;
+
```

Comment:
Might be worth adding a little bit of spacing between the list items themselves, I'm a bit worried about the links being too close together, particularly on mobile.